node_modules/col-rest/lib/rest/activities.js

```diff
@@ -75,6 +75,38 @@ module.exports = function(client) {
     client.jsonPost(requestUrl, activityTypeUpdates, null, callback);
   };
 
+  /**
+   * Get activities for a user ID
+   *
+   * @param  {Course}         course                            The Canvas course in which the user is interacting with the API
+   * @param  {Number}         userId                            The SuiteC id of the user for which activities should be returned
+   * @param  {Function}       callback                          Standard callback function
+   * @param  {Object}         callback.err                      An error that occurred, if any
+   * @param  {Object}         callback.body                     The JSON response from the REST API
+   * @param  {Response}       callback.response                 The response object as returned by requestjs
+   * @see col-activities/lib/rest.js for more information
+   */
+  client.activities.getActivitiesForUserId = function(course, userId, callback) {
+    var requestUrl = client.util.apiPrefix(course) + '/activities/user/' + client.util.encodeURIComponent(userId);
+    client.request(requestUrl, 'GET', null, null, callback);
+  };
+
+  /**
+   * Get activities for an asset ID
+   *
+   * @param  {Course}         course                            The Canvas course in which the user is interacting with the API
+   * @param  {Number}         userId                            The id of the asset for which activities should be returned
```

Comment:
@paulkerschen All looks good with this PR. Small correction: 'userId' (above) should be 'assetId'.